funcs/func_global.c

```diff
@@ -145,6 +158,18 @@ static struct ast_custom_function global_function = {
 	.write = global_write,
 };
 
+static int global_delete_write(struct ast_channel *chan, const char *cmd, char *data, const char *value)
+{
+	pbx_builtin_setvar_helper(NULL, data, NULL);
```

Comment:
This function returns an int, usually 0, so could be simplified to just:

`return pbx_builtin_setvar_helper(NULL, data, NULL);`